<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1090</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>941</width>
     <height>491</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="volkswagen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>85c020adabb7db9484d62268305c64bc.jpg</normaloff>85c020adabb7db9484d62268305c64bc.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>120</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rols_royce">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>rolse_royce.jpg</normaloff>rolse_royce.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="Ferrari">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>ferari.png</normaloff>ferari.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>120</width>
     <height>120</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="lamborghini">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>lamborghini.png</normaloff>lamborghini.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="porshe">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>porshe.png</normaloff>porshe.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>150</width>
     <height>150</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="bently">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>111</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>bently.png</normaloff>bently.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>160</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="srav">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>161</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>srav1.png</normaloff>srav1.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="korz">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>korz.jpg</normaloff>korz.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>180</width>
     <height>180</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="trade_in">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>10</y>
     <width>151</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>TRADE_IN.png</normaloff>TRADE_IN.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>105</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>play.jpg</normaloff>play.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>110</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="plus">
   <property name="geometry">
    <rect>
     <x>910</x>
     <y>10</y>
     <width>131</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 10px;
background: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>lus.jpg</normaloff>lus.jpg</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>200</width>
     <height>110</height>
    </size>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
